src/collections/_documentation/platforms/python/generic.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Serverless
```

Comment:
I would keep this named serverless (I mean the filename). We have this naming pattern established with WSGI already, which is also a "generic" integration.

Reply:
Suggestion taken